### PR TITLE
49 ta fb profile cards

### PR DESCRIPTION
**Pull request by FadiBaker92**
2 shared cards created but didn't render the partials yet:
-  User card
-  Program info card

Screenshot 2024-10-14 140734

### CODE REVIEW

**Comment by FadiBaker92** [comment]
Screenshot 2024-10-14 191321
Screenshot 2024-10-14 191337
updated screen shots

File: app/views/dashboard/_program_card.html.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  <img class="w-full h-[5.538]" src="<%= asset_path("sample_img/dpi_building.png") %>" alt="Dpi building">
+  <div class="px-6 py-4">
+    <p class="text-xs text-gray-400">Program name</p>
+    <div class="text-base font-semibold mb-2 mt-2"> <%= current_user.cohort.cohort_name %> </div> 
+    <p class="text-[0.875rem] text-wrap text-[#898989] font-400 mb-2">
+      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+    </p>
+    <button class="text-xs w-[8.75rem] h-[32px] bg-[#202020] text-[#FFFFFF] border border-transparent rounded-md hover:bg-transparent hover:text-[#202020] hover:border-[#202020]"
+            onclick="location.href='#'">
+      Apply Pairing
```
Comment (MayaS1111) [inline app/views/dashboard/_program_card.html.erb:11]: No button for now. You can just comment it out

**Review by MayaS1111 — CHANGES_REQUESTED** [review]

**Review by APatel-AI — APPROVED** [review]
Looks great!

**Review by MayaS1111 — APPROVED** [review]
Great work, thanks fadiiiii!

**Pull request merged**